OrbitCore/OrbitModule.cpp

```diff
@@ -134,58 +135,39 @@ std::string FindSymbols(const std::string& module_path) {
   return module_path;
 }
 
-//-----------------------------------------------------------------------------
-bool Pdb::LoadPdb(const char* a_PdbName) {
-  m_FileName = FindSymbols(a_PdbName);
-  m_Name = Path::GetFileName(m_FileName);
-  m_LoadedModuleName = a_PdbName;
+bool Pdb::LoadFunctions(const std::string& file_name) {
+  m_FileName = file_name;
```

Comment:
The call to "FindSymbols" was dropped ( m_FileName = FindSymbols(a_PdbName); ).  This will prevent loading of symbols from separate files.  Is this a limitation of the new approach?  We need to preserve that functionality...

Reply:
it was not dropped.- see below.

Reply:
Ah ok, there is still an issue, m_FileName and m_LoadedModuleName should not be the same in the case where separate debug symbols were found.  m_FileName should be assigned the result of FindSymbols, but m_LoadedModuleName should be the name of the module.